DigitalLearningSolutions.Data.Tests/Services/DiagnosticAssessmentServiceTests.cs

```diff
@@ -178,12 +153,25 @@ public void Get_diagnostic_assessment_should_return_null_if_archived_date_is_not
         }
 
         [Test]
-        public void Get_diagnostic_assessment_should_return_null_if_diag_status_and_status_and_is_assessed_are_false()
+        public void Get_diagnostic_assessment_should_return_null_if_section_diagAssessPath_is_null()
+        {
+            // When
+            const int customisationId = 22206;
+            const int candidateId = 210962;
+            const int sectionId = 1920;
+            var result = diagnosticAssessmentService.GetDiagnosticAssessment(customisationId, candidateId, sectionId);
+
+            // Then
+            result.Should().BeNull();
+        }
+
+        [Test]
+        public void Get_diagnostic_assessment_should_return_null_if_all_diagStatus_are_false()
         {
             // Given
-            const int customisationId = 1530;
-            const int candidateId = 23573;
-            const int sectionId = 74;
+            const int customisationId = 4831;
+            const int candidateId = 11;
+            const int sectionId = 75;
```

Comment:
Why did we need to change this data? 🙂

Reply:
The `DiagStatus` values for the old case were all 0, so we couldn't be sure that the reason the function return `null` was because of the `null` `diagAssessPath`, when it could have just been because of the `DiagStatus` values 🙂

Reply:
I see, that makes sense 🙂